app/models/user.server.ts

```diff
@@ -3,8 +3,51 @@ import bcrypt from "bcryptjs";
 
 import { prisma } from "~/db.server";
 
+const algorithm = "aes-256-ctr";
+if (!process.env.ENCRYPTION_KEY_1 || !process.env.ENCRYPTION_KEY_2) {
+  throw new Error("Please setup encryption keys");
+}
+const secretKey = process.env.ENCRYPTION_KEY_1 + process.env.ENCRYPTION_KEY_2;
+
 export type { User } from "@prisma/client";
 
+export async function encryptAllSecrets() {
+  const crypto = await import("node:crypto");
+  const algorithm = "aes-256-ctr";
+  if (!process.env.ENCRYPTION_KEY_1 || !process.env.ENCRYPTION_KEY_2) {
+    throw new Error("Please setup encryption keys");
+  }
+  const secretKey = process.env.ENCRYPTION_KEY_1 + process.env.ENCRYPTION_KEY_2;
+
+  const usersWithUnecryptedLnmSecrets = await prisma.password.findMany({
+    where: {
+      iv: "",
+    },
+  });
+
+  const results = usersWithUnecryptedLnmSecrets.forEach(async (data) => {
+    const iv = crypto.randomBytes(16);
+    const cipher = crypto.createCipheriv(algorithm, secretKey, iv);
+    const encrypted = Buffer.concat([
+      cipher.update(data.lnmSecret),
+      cipher.final(),
+    ]);
+
+    const encryptedLnmSecret = encrypted.toString("hex");
+    const user = await prisma.password.update({
+      where: {
+        userId: data.userId,
+      },
+      data: {
+        lnmSecret: encryptedLnmSecret,
```

Comment:
my concern with overwriting the existing `lnmSecret` is if there's some problem with the encryption/decryption then we'll lose this record. should we just add a new field `encryptedLnmSecret` instead?

Reply:
and then on a second endpoint, remove all existing `lnmSecret`s once we verify decryption is working?

Reply:
Yup that is a safer way. 
I did test this locally all is good but we cannot recover is something goes wrong. 
I will use a separate column